src/main/java/com/fjb/sunrise/mappers/UserMapper.java

```diff
@@ -1,7 +1,10 @@
 package com.fjb.sunrise.mappers;
 
+import com.fjb.sunrise.dtos.requests.RegisterRequest;
+import com.fjb.sunrise.models.User;
 import org.mapstruct.Mapper;
 
 @Mapper(componentModel = "spring")
 public interface UserMapper {
+    User toEntity(RegisterRequest request);
```

Comment:
consider change the name from toEntity -> toUserEntity for more clear, because we have many different mappers

Reply:
but this is the user mapper